charts/dns_canary/templates/cronjob.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+spec:
+  schedule: {{ .Values.schedule | quote }}
```

Comment:
There is no `schedule` in the values file. Add it to the values file with a reasonable default schedule. Probably every 20 minutes by default.